### PR TITLE
offline operator learning + pg3 config files

### PR DESCRIPTION
**Pull request by Abraham-Mitchell**
offline pg3 config file

### CODE REVIEW

File: scripts/configs/offline_config.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# PG4 experiments.
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:1]: retitle

Comment (tomsilver) [inline scripts/configs/offline_config.yaml:1]: please also rename this file to `pg3_offline_config.yaml`

File: scripts/configs/offline_config.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: {}
+  plan-only:
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:7]: let's add another approach for PG3 with given operators. I think it should be:

```
pg3-given-ops:
    NAME: "pg3"
    FLAGS:
        strips_learner: oracle
```

File: scripts/configs/offline_config.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    NAME: "oracle"
+    FLAGS: {}
+    
+ENVS:
+  deliver:
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:12]: delivery*

File: scripts/configs/offline_config.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    NAME: "pddl_easy_delivery_procedural_tasks"
+    FLAGS:
+      explore_random_options:
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:15]: these flags look like they are for the online learning experiments, not the offline learning ones -- please remove and update. also for any flags that are not specific to an environment, let's put them in the general `FLAGS` section below (currently on line 30)

File: scripts/configs/offline_config.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      max_initial_demos: 1
+      num_train_tasks: 10
+      num_test_tasks: 10
+      max_num_steps_interaction_request: 10
+  spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS:
+      explore_random_options:
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:23]: same note here about offline vs online flags

File: scripts/configs/offline_config.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      max_num_steps_interaction_request: 10
+
+ARGS: []
+FLAGS:
+  strips_learner: oracle
+  sampler_learner: oracle
+  num_train_tasks: 50
+BRANCH: master
+START_SEED: 456
+NUM_SEEDS: 10
```
Comment (tomsilver) [inline scripts/configs/offline_config.yaml:36]: this is just a convention, but let's keep a single new line at the end of the file

**Review by tomsilver — COMMENTED** [review]
@Abraham-Mitchell good start, here are some comments! also note that I clicked the "update branch with master" button. so before you start making changes, first do `git pull origin abraham-stop-pg3-relearning` to synchronize your local branch with this github remote branch

**Review by tomsilver — APPROVED** [review]
great work! let's try it!

File: scripts/configs/pg3_offline_config.yaml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+  delivery:
+    NAME: "pddl_easy_delivery_procedural_tasks"
+    FLAGS: {}
+  spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS: {}
+
+ARGS: []
+FLAGS:
+  strips_learner: oracle
```
Comment (ronuchit) [inline scripts/configs/pg3_offline_config.yaml:25]: is this shared? won't that defeat the purpose of having it on only some of the approaches above? @tomsilver

Comment (tomsilver) [reply scripts/configs/pg3_offline_config.yaml:25]: crap yeah thanks good catch!

**Pull request merged**